examples/mixer_demo.rs

```diff
@@ -30,10 +32,13 @@ fn demo(filename: &Path) {
 
     let frequency = 44100;
     let format = AUDIO_S16LSB; // signed 16 bit samples, in little-endian byte order
-    let channels = 2; // Stereo
+    let channels = DEFAULT_CHANNELS; // Stereo
     let chunk_size = 1024;
     sdl2::mixer::open_audio(frequency, format, channels, chunk_size).unwrap();
-    sdl2::mixer::allocate_channels(0);
+
+    // Number of mixing channels available for sound effect `Chunk`s to play
+    // simultaneously.
+    sdl2::mixer::allocate_channels(4);
```

Comment:
Does this mean that you can play 4 sound effects + the usual music with that? Is that why iy wouldn't work with `sdl2::mixer::allocate_channel(0)`?